digitalocean/resource_digitalocean_certificate.go

```diff
@@ -21,6 +22,9 @@ func resourceDigitalOceanCertificate() *schema.Resource {
 			State: schema.ImportStatePassthrough,
 		},
 
+		SchemaVersion: 1,
+		MigrateState:  resourceCertificateMigrateState,
```

Comment:
This looks like a 0.11 style migration. I think we want something closer to this:

```
		StateUpgraders: []schema.StateUpgrader{
			{
				Type:    resourceDigitalOceanCertificateV0().CoreConfigSchema().ImpliedType(),
				Upgrade: migrateCertificateStateV0toV1,
				Version: 0,
			},
		},
```

https://www.terraform.io/docs/extend/resources/state-migration.html#terraform-v0-12-sdk-state-migrations